.github/workflows/pull-request.yml

```diff
@@ -130,8 +130,7 @@ jobs:
         # Use groups to avoid spamming the web interface. Pay attention
         # to COMMAS. Don't use a single big group so a single failure
         # does not block all other builds.
-        platform: [imx8ulp,
-                   rn rmb,
+        platform: [rn rmb,
```

Comment:
can you please add `imx8m` here?

Reply:
can do but it'll end up being temporary as XTOS will be deprecated on MP as well soon